src/ui/server/controllers/insurance/feedback/feedback-form/index.ts

```diff
@@ -95,9 +102,20 @@ const post = async (req: Request, res: Response) => {
         [OTHER_COMMENTS]: feedback[OTHER_COMMENTS],
       } as InsuranceFeedbackVariables;
 
+      const feedbackVariables = {
+        [SATISFACTION]: feedback[SATISFACTION] ?? '',
+        [IMPROVEMENT]: feedback[IMPROVEMENT],
+        [OTHER_COMMENTS]: feedback[OTHER_COMMENTS],
+        [SERVICE]: INSURANCE,
+        [REFERRAL_URL]: referralUrl?.toString(),
+        [PRODUCT]: SERVICE_NAME,
+      };
+
+      const saveResponse = await api.keystone.feedback.create(feedbackVariables);
```

Comment:
Not a blocker, but we should just have 1 API call "create feedback" that creates the feedback and sends the email. Can improve after merge

Reply:
will improve this after merge